include/vcpkg/base/jsonreader.h

```diff
@@ -279,8 +277,8 @@ namespace vcpkg::Json
 
     struct StringDeserializer final : IDeserializer<std::string>
     {
-        virtual StringView type_name() const override { return type_name_; }
-        virtual Optional<std::string> visit_string(Reader&, StringView sv) override { return sv.to_string(); }
+        StringView type_name() const override { return type_name_; }
```

Comment:
Now that this is not `virtual` anymore, it can be marked as `constexpr`. Same elsewhere.

Reply:
No. It is still virtual. This is implied by `override`.

Reply:
Whoops, I should have looked past the function name 😂